### DISCUSSION TITLE
what is .gitignore file ?

### DISCUSSION THREAD

**Discussion by theloser404** [community/community#28002] [Other Feature Feedback, Questions, & Ideas] [answered]
what is .gitignore file ?

**Comment by iamfoysal** [comment, marked as answer]
.gitignore protects the repo from unwanted files

see more: [Doc](https://git-scm.com/docs/gitignore)

**Reply by theloser404** [reply]
Thanks

**Reply by chinnu0209** [reply]
.gitignore are excluded files and folders from source-control tracking via git they will simply ignore . files .
